pkg/volume/flexvolume/driver-call.go

```diff
@@ -235,3 +241,29 @@ func handleCmdResponse(cmd string, output []byte) (*DriverStatus, error) {
 
 	return &status, nil
 }
+
+// getDriverCapabilities returns the reported capabilities as returned by driver's init() function
+func getDriverCapabilities(plugin *flexVolumePlugin) (*driverCapabilities, error) {
+	call := plugin.NewDriverCall(initCmd)
```

Comment:
@verult if this is something we have to support, how do you do that with a regular driver? I don't we have to support any kind of "self recast" of the driver. If a driver has to attach in some circumstances and not other, it *is* attache *able*, just with a no-op `Attach` call in these cases. A second point against init being called by us is that it is part of the VolumePlugin` interface so we really should not have any call to `init` outside `FlexVolumeVolume.Init()`, and never this one either.